index.d.ts

```diff
@@ -37,6 +38,12 @@ declare const findUp: {
 
 		console.log(await findUp(['rainbow.png', 'unicorn.png']));
 		//=> '/Users/sindresorhus/unicorn.png'
+
+		console.log(await findUp(async directory => {
```

Comment:
Should this example be moved into the comment for the matcher variant?

Reply:
Yes